src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, ac) {
+      return a + ac;
+    },
+
+    subtract(a, ac) {
+      return a - ac;
+    },
+
+    multiply(a, ac) {
```

Comment:
```suggestion
    multiply(value) {
```
no need to pass this.result you have access to this anyway